### PR TITLE
Added sneaking option to controll states

### PR DESCRIPTION
**Pull request by Meldiron**
- added option 'sneak' to setControllState,
- changed api documentation, added this option there

### CODE REVIEW

File: lib/plugins/physics.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ function inject (bot) {
     left: false,
     right: false,
     jump: false,
-    sprint: false
+    sprint: false,
+    sneak: false,
```
Comment (plexigras) [inline lib/plugins/physics.js]: eslint doesn't like the trailing commas, you will need to remove it to confirm to the code style rules

Comment (plexigras) [inline lib/plugins/physics.js]: eslint doesn't like the trailing commas, you will need to remove it to confirm to the code style rules.

Comment (plexigras) [inline lib/plugins/physics.js]: eslint doesn't like the trailing commas, you will need to remove it to confirm to the code style rules

**Comment by rom1504** [comment]
Thanks !

**Pull request merged**